packages/react-core/src/components/Wizard/examples/WizardWithDrawer.tsx

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import {
+  Button,
+  DrawerActions,
+  DrawerCloseButton,
+  DrawerColorVariant,
+  DrawerHead,
+  DrawerPanelContent,
+  Wizard
+} from '@patternfly/react-core';
+
+export const WizardWithDrawer: React.FunctionComponent = () => {
+  /* eslint-disable @typescript-eslint/no-unused-vars */
+  const [isOpen, setIsOpen] = React.useState(false);
+  const [isDrawerExpanded, setIsDrawerExpanded] = React.useState(false);
+  const [sectionGray, setSectionGray] = React.useState(false);
+  const [panelGray, setPanelGray] = React.useState(true);
+  const [contentGray, setContentGray] = React.useState(false);
+  /* eslint-enable @typescript-eslint/no-unused-vars */
+
+  const drawerRef = React.useRef<HTMLSpanElement>(null);
+
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const onExpand = () => {
+    drawerRef.current && drawerRef.current.focus();
+  };
```

Comment:
@jenny-s51 was the original intention of this onExpand method that once the drawer opens, the drawer (or rather the drawer's close button) is given focus?

Reply:
Nice catch @thatblindgeye ! Yes, the drawer's close button should definitely be given focus once the drawer opens. 

I've been investigating... It looks like `onExpand` is never called because it can only be set on the `<Drawer>` component to work correctly, and `<Drawer>` is no longer declared in the examples, since that logic is abstracted via the `hasDrawer` prop. 

Since `onExpand` isn't being called in these examples, I think it would make sense to remove these unused function declarations for now, and open a bug to fix the focus issue since it likely won't be a straightforward fix!